release_bot/pypi.py

```diff
@@ -35,9 +35,13 @@ def __init__(self, configuration, git):
         self.logger = configuration.logger
         self.git = git
 
+
     def latest_version(self):
         """Get latest version of the package from PyPi or 0.0.0"""
-        response = requests.get(url=f"{self.PYPI_URL}{self.conf.repository_name}/json")
+        if self.conf.pypi_project != None:
+            response = requests.get(url=f"{self.PYPI_URL}{self.conf.pypi_project}/json")
```

Comment:
we should utilize only one, not have if/else spread across the whole codebase, just store the value in self.conf.pypi_project and use that one